src/modules/removeTask.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/extensions
```

Comment:
As mentioned above, kindly avoid disabling linter checks inline. Remember to remove all other occurrences of it.